src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -732,6 +735,30 @@ public async Task HandleDidChangeTextDocumentNotification(ScriptFile[] changedFi
             }
         }
 
+        /// <summary>
+        /// Handle the file close notification
+        /// </summary>
+        /// <param name="uri"></param>
+        /// <param name="scriptFile"></param>
+        /// <param name="eventContext"></param>
+        /// <returns></returns>
+        public Task HandleDidSaveTextDocumentNotification(
+            string uri,
+            ScriptFile scriptFile,
+            EventContext eventContext)
+        {
+            try
+            {
+                // TODO: need to do something here.
```

Comment:
Is this needed anymore?

Also did you ever look into using this event? Was there some reason you couldn't use it?

Reply:
It's generally not needed, I just did not know what to do with this event on the ADS side as its not implemented in any project that I was involved with.